src/lib/Nav.svelte

```diff
@@ -133,24 +137,40 @@
   {#if isMobileMenuOpen}
     <div class="absolute left-0 right-0 top-[84px] w-[100%] bg-[#0b383c] lg:hidden">
       <div class="flex flex-col w-full px-8 py-4 space-y-4">
+<<<<<<< HEAD
+        <!-- Mobile Search -->
+=======
+>>>>>>> upstream/main
```

Comment:
some leftover comments/tags like this left by the merge conflicts to clean up.
Also your PR seems to break on prod build for now. mind checking it _(sorry for the inconvenience)_